src/main/java/org/isf/examination/rest/ExaminationController.java

```diff
@@ -81,38 +85,15 @@ public ResponseEntity<Boolean> newPatientExamination(@RequestBody PatientExamina
         if (patient == null) {
             throw new OHAPIException(new OHExceptionMessage("Patient does not exist."));
         }
-        if (newPatientExamination.getPex_height() < 0 || newPatientExamination.getPex_height() > 250) {
-        	throw new OHAPIException(new OHExceptionMessage("The size should be between 0 and 250."));
-        }
-        if (newPatientExamination.getPex_weight() < 0.0 || newPatientExamination.getPex_weight() > 200.0) {
-        	throw new OHAPIException(new OHExceptionMessage("The weight should be between 0 and 200."));
-        }
-        if (newPatientExamination.getPex_ap_min() < 80) {
-        	throw new OHAPIException(new OHExceptionMessage("The minimum blood pressure must be at least 80."));
-        }
-        if (newPatientExamination.getPex_ap_min() > newPatientExamination.getPex_ap_max() ) {
-        	throw new OHAPIException(new OHExceptionMessage("The minimum blood pressure must be lower than the maximum blood pressure."));
-        }
-        if (newPatientExamination.getPex_ap_max() > 120) {
-        	throw new OHAPIException(new OHExceptionMessage("The maximum blood pressure must be lower than 120."));
-        }
-        if (newPatientExamination.getPex_hr() < 0 || newPatientExamination.getPex_hr() > 240 ) {
-        	throw new OHAPIException(new OHExceptionMessage("Heart rate should be between 0 and 240."));
-        }
-        if (newPatientExamination.getPex_temp() < 30.0 || newPatientExamination.getPex_temp() > 50.0) {
-        	throw new OHAPIException(new OHExceptionMessage("The temperature should be between 30 and 50."));
-        }
-        if (newPatientExamination.getPex_sat() < 50.0 || newPatientExamination.getPex_temp() > 100.0) {
-        	throw new OHAPIException(new OHExceptionMessage("The saturation should be between 50 and 100."));
+        validateExamination(newPatientExamination);
+        if (newPatientExamination.getPex_diuresis_desc() == null) {
+        	newPatientExamination.setPex_diuresis_desc(Diurese.physiological);
```

Comment:
this is not correct, every parameter could be null, except Height and Weight that are compulsory.

The check here is to verify that the passed string is within the allowed ones (which are mapped and hardcoded)

Why we don't leverage the validation in core?